src/components/PeopleFilters/PeopleFilters.tsx

```diff
@@ -0,0 +1,150 @@
+import { Link, useSearchParams } from 'react-router-dom';
+import classNames from 'classnames';
+
+export const PeopleFilters = () => {
+  const centuriesList = [16, 17, 18, 19, 20];
+  const [searchParams, setSearchParams] = useSearchParams();
+  const query = searchParams.get('query') || '';
+  const sex = searchParams.get('sex') || 'all';
+  const centuries = searchParams.getAll('centuries') || [];
+
+  const handleQuery = (ch: string) => {
+    const params = new URLSearchParams(searchParams);
+
+    params.set('query', ch);
+
+    const check = { query: ch };
+
+    setSearchParams(params);
+    if (!check.query) {
+      params.delete('query');
+      setSearchParams(params);
+    }
+  };
+
+  const handleSex = (gender: string) => {
+    const params = new URLSearchParams(searchParams);
+
+    params.set('sex', gender);
+    if (gender === 'all') {
+      params.delete('sex');
+    }
+
+    return params.toString();
+  };
+
+  const handleCenturies = (century: string) => {
+    const params = new URLSearchParams(searchParams);
+
+    const newCenturies = centuries.includes(century.toString())
+      ? centuries.filter(cent => cent !== century.toString())
+      : [...centuries, century];
+
+    params.delete('centuries');
+    if (century !== '') {
+      newCenturies.forEach(cen => params.append('centuries', cen.toString()));
+    }
+
+    return params.toString();
+  };
+
+  const clear = () => {
```

Comment:
I think it is better to name this function a more descriptive name, rather than clear. As for example handleResetAll or onReset 🤷‍♂️